deepchem/models/tests/test_optimizers.py

```diff
@@ -1,5 +1,7 @@
+import deepchem as dc
 import deepchem.models.optimizers as optimizers
 import unittest
+from deepchem.models.torch_models import kfac_optimizer
```

Comment:
This import statement depends on torch module. It is better to keep it's scope local to the test method. In this case, it will `test_KFAC` method